modules/tracking/test/test_trackers.cpp

```diff
@@ -87,6 +87,8 @@ class TrackerTest
 
   void distanceAndOvrerlapTest();
 
+  void gtrMemoryTest();
```

Comment:
TrackerTest is a universal test for every tracker. That means this GOTURN check will be performed for every other tracker test. So please make it as a standalone test:
```cpp
// https://github.com/opencv/opencv_contrib/issues/2589
TEST(GOTURN, memory_usage)
{
  cv::Rect2d roi;
  cv::Mat frame;
  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
  // ...
}

```

Reply:
OK